### PR TITLE
Everything is finally done regarding my personal items, i sprited and…

### PR DESCRIPTION
**Pull request by LunaPumpkin**
… coded them all and lost my mind a couple times cuz i had to re do it 3 times, then one of my files locked up, yadda yadda, fixed that, re did everything. EVERYTHING IS FUNCTIONING NORMALLY. ive added my personal items of a set of tails, ears, and a robe to go with them

:cl:
add: Added new things

imageadd: added some icons and images
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1516,3 +1516,23 @@
...
+/datum/sprite_accessory/mam_tails/lunasune
+	name = "lunasune"
+	icon_state = "lunasune"
+	extra = TRUE
+	ckeys_allowed = list("invader4352")
+
+/datum/sprite_accessory/mam_tails_animated/lunasune
+	name = "lunasune"
+	icon_state = "lunasune"
+	extra = TRUE
```
Comment (deathride58) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]: These are duplicates, they're already defined in sprite_accessories.dm in modular_citadel

**Review by deathride58 — DISMISSED** [review]
Needs approval from @LetterJay for the following:
One whitelisted tail for ckey invader4352
One whitelisted ears for ckey invader4352
Cosmetic divine robes for ckey invader4352

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 2]
```diff
@@ -397,3 +397,11 @@
 	desc = "Comfy Lucky Jackboots with the word Luck on them."
 	item_state = "luckyjack"
 	icon_state = "luckyjack"
+
+/obj/item/clothing/under/lunasune
+	name = "Divine Robes"
+	icon = 'icons/obj/custom.dmi'
+	icon_override = 'icons/mob/custom_w.dmi'
```
Comment (Poojawa) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm:404]: should probably change this to `alternate_worn_icon`

**Review by Poojawa — CHANGES_REQUESTED** [review]
Make sure you clean up the 🆑  stuff so the changelog isn't filled with nonsense.

File: modular_citadel/code/modules/custom_loadout/custom_items.dm (hunk 2 above)
Comment (LunaPumpkin) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm:404]: I don't know how to do either of those things, it took me the help of 3 contributors to do it all.

Comment (LunaPumpkin) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm:404]: Fixed CL

Comment (LunaPumpkin) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm:404]: after exploring, I figured both things out, id like to leave it as is

**Review by LetterJay — APPROVED** [review]

**Pull request merged**